features/step_definitions/front_office/renewal_steps.rb

```diff
@@ -449,30 +390,18 @@
   @renewals_app.contact_email_page.submit
   @renewals_app.contact_manual_address_page.submit(
     house_number: "1",
-    address_line_one: "Test lane",
-    address_line_two: "Testville",
-    city: "Teston",
-    country: "Slovakia"
+    address_line_one: "Via poerio",
+    address_line_two: "Via Poerio",
+    city: "Rome",
+    postcode: "00152", # required, as tests currently fail without a postcode
+    country: "Italy"
   )
   @renewals_app.check_your_answers_page.submit
   @renewals_app.declaration_page.submit
   @renewals_app.registration_cards_page.submit
   @renewals_app.payment_summary_page.submit(choice: :card_payment)
-  @renewals_app.worldpay_card_choice_page.submit
-  # finds today's date and adds another year to expiry date
-  time = Time.new
-
-  @year = time.year + 1
-
-  @renewals_app.worldpay_card_details_page.submit(
-    card_number: "6759649826438453",
-    security_code: "555",
-    cardholder_name: "3d.authorised",
-    expiry_month: "12",
-    expiry_year: @year,
-    postcode: "90210"
-  )
-  @renewals_app.worldpay_secure_page.submit
+  sleep(1)
```

Comment:
Do we still need this `sleep`?

Reply:
No we don't - well spotted. I've removed this.